response.go

```diff
@@ -72,6 +73,11 @@ func (meta *ColumnMetaData) DecodeMsgpack(d *msgpack.Decoder) error {
 	return nil
 }
 
+const (
+	keySQLInfoRowCount         = 0x00
```

Comment:
These two also seem like they should be in go-iproto

Reply:
Updated.